### PR TITLE
Fix moe_normalization_mode on Mixtral and Arctic

### PR DESCRIPTION
**Pull request by lkm2835**
Fix mixtral and arctic normalization modes (RENORM -> NONE)

[Normalization Modes](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/mixtral#normalization-modes)

- 0 (NONE) corresponds to: scales = topk(softmax(routing values))
- 1 (RENORM) corresponds to: scales = softmax(topk(routing values))

[Mixtral HF Modeling: topk(softmax(router values))](https://github.com/huggingface/transformers/blob/0de15c988b0d27758ce360adb2627e9ea99e91b3/src/transformers/models/mixtral/modular_mixtral.py#L197-L198) = (NONE)

[Arctic HF Modeling: topk(softmax(router values))](https://huggingface.co/Snowflake/snowflake-arctic-instruct/blob/main/modeling_arctic.py#L973-L974) = (NONE)

[GraniteMoe HF Modeling: softmax(topk(router values))](https://github.com/huggingface/transformers/blob/main/src/transformers/models/granitemoe/modeling_granitemoe.py#L323-L324) = (RENORM)

### CODE REVIEW

**Comment by djns99** [comment]
Hi @lkm2835, thanks you for your interest in contributing to TensorRT-LLM.

You'll notice in the Mixtral and Arctic implementations you link they use:

```
        routing_weights = F.softmax(router_logits, dim=1, dtype=torch.float)
        routing_weights, selected_experts = torch.topk(routing_weights, self.top_k, dim=-1)
        routing_weights /= routing_weights.sum(dim=-1, keepdim=True)
```

The extra division makes this equivalent to `softmax(topk(routing values))`, i.e. RENORM mode

Please let us know if you notice any other issues and we would love to consider them. For now, I will close this issue
